### PR TITLE
Romain

### PR DESCRIPTION
**Pull request by Fostrankill**

### CODE REVIEW

**Review by cartige — DISMISSED** [review]

File: frontend/src/pages/Home/Home.jsx [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
 import { Swiper, SwiperSlide } from "swiper/react";
 import { Navigation, Pagination, Mousewheel, Keyboard, FreeMode } from "swiper";
+import YouTube from "react-youtube";
 import Registration from "@components/Registration/Registration";
 import NavBar from "../../components/NavBar/NavBar";
 /* eslint-disable import/no-unresolved */
 import "swiper/css";
 import "swiper/css/pagination";
 import "swiper/css/navigation";
+
```
Comment (LoicBrassart) [inline frontend/src/pages/Home/Home.jsx:13]: Why add this empty space ?

File: frontend/src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
+  useEffect(() => {
+    axios
+      .get("http://localhost:5000/videos")
+      .then(({ data }) => {
+        setVideos(data);
+      })
+      .catch((err) => {
+        console.warn("------------", err);
+      });
```
Comment (LoicBrassart) [inline frontend/src/pages/Home/Home.jsx]: Even though ESLint lets you do this, it has no meaning, remove it

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Remove useless changes and it's okay!

**Review by Fostrankill — COMMENTED** [review]
Add Modification

**Review by cartige — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**